finclab/my-wealth-frontend
# Title: convert the saving page to react

#### What does this PR do?
- This PR implements the conversion of the savings page to react components

#### Description of Task to be completed?
- Convert savings page to react
- fix failing test in sign up form component

#### How should this be manually tested?
- Clone the repo by running `git clone <github Url>`
- Cd into the directory and run `npm install` to install all dependencies
- In the directory run `npm run start` to start the react application.
- Visit `http://localhost:3000/savings` on your browser to see the application
 
#### Any background context you want to provide?
<img width="1440" alt="Screen Shot 2019-04-24 at 3 46 33 PM" src="https://user-images.githubusercontent.com/32917511/56668874-2d2d6780-66a8-11e9-8a93-5f71db0c290c.png">


#### What are the relevant pivotal tracker stories?(if applicable)
[#165565237](https://www.pivotaltracker.com/story/show/165565237)
